php/PhpParser.g4

```diff
@@ -489,7 +492,7 @@ expression
     | (Include | IncludeOnce) expression                       #SpecialWordExpression
     | (Require | RequireOnce) expression                       #SpecialWordExpression
 
-    | Static? Function '&'? '(' formalParameterList ')' lambdaFunctionUseVars? blockStatement  #LambdaFunctionExpression
+    | lambdaFunctionExpr                                       #LambdaFunctionExpression
```

Comment:
Please align spaces with other alternative labels

Reply:
should aligned now